crates/fuel-core/src/service/genesis.rs

```diff
@@ -4,11 +4,13 @@ use crate::{
 };
 use anyhow::anyhow;
 use fuel_core_chain_config::{
```

Comment:
This file seems unfinished=) Do we need to review it, or do you plan to finish it in another PR?

Reply:
The changes to `genesis.rs` are placeholders to appease the CI. They undergo heavy changes in https://github.com/FuelLabs/fuel-core/pull/1519 and can be ignored here.